test/test_datasets.py

```diff
@@ -1,5 +1,17 @@
 import unittest
 import pandas as pd
+import tempfile
+import requests
+import tarfile
+import gzip
+import csv
+import os
+
+from io import BytesIO, StringIO
```

Comment:
`StringIO` isn't used according to Codacy (on the PR checks, click on "details" next to Codacy check, you'll get the full details ;) )

Reply:
Thanks a lot !